refs/high-level-conceptualization/README.md

```diff
@@ -109,12 +116,27 @@ States concern different qualities, such as (not limited to):
 
 Legal grounds also have states indicating if the data MUST or CAN be kept/deleted.
 
-Rights requests also have states (e.g., received, denied, granted,...) – states relative to a particular request.
+Rights Requests also have states with regards to:
+
+- **Their treatment** by the Data Consumers / Organisation (received, viewed/under review, treated)
+- **Acceptance** (accepted, partially accepted, rejected)
+- **Fulfillment** of the Data Subject's need (data accessed, data modified, data deleted)
+
+The system keeps a record of every status change and the corresponding timestamp allowing to build a timeline.
 
 Also, rights have states, meaning that in a particular state, a particular type of rights request MUST be rejected. For example, during mandatory keeping a DELETE request from the user MUST be rejected.
 
 > _The information provided here is just for illustrative purposes and does not imply any definitive naming or semantics of actual states._
 
+## DATA CAPTURE – VERSIONING
+
+Data Capture Fragments are subject to versioning. A particuler data fragment can evolve as a result of:
+- **Changes** by Data Subject during submission
+- **Rights Requests** by Data Subject asking to modify the data
+- **Modifications** by Data Consumers and DPOs (when the new data is collected offline from the Data Subject)
```

Comment:
Also modifications BY data subjects not related to a rights request, if the access to data is provided through the UI (changing the phone number on a user's profile page, request for additional data by data consumer, etc.).